eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/main.tsp

```diff
@@ -32,7 +32,4 @@ enum Versions {
 }
 
 @@clientLocation(Bars.get, "Bar");
-@@clientLocation(Bars.update, "Bar");
-
-#suppress "deprecated"
-@@flattenProperty(BarSettingsResource.flattenedNestedProperty);
```

Comment:
the namespace of this decorator changed to `Azure.ClientGenerator.Core.Legacy`, instead of removing this, we should update the namespace here.